src/ComplexMarkdownParser.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Spatie\YamlFrontMatter;
+
+use Symfony\Component\Yaml\Yaml;
+
+/**
```

Comment:
The class will become some much lighter if we remove all the comments. Using good names will already make clear what is going on.

Could you remove all the comments?

Reply:
Will do! Fill fix the other conversations as well!